src/Roadmap.tsx

```diff
@@ -197,41 +173,106 @@ export function Roadmap() {
                                 item: 'Burn and Holder Verification Bot',
                             },
                         ]}
+                    />
+
+                    <RoadmapBundle
+                        label={'Q2 2022'}
+                        items={[
+                            {
+                                item: 'Trending Collections Bot',
+                                complete: true,
+                            },
+                            {
+                                item: 'Slug Sales Bot',
+                                complete: true,
+                            },
+
+                        ]}
                         initiallyExpanded={true}
```

Comment:
Lets make these all unexpanded except for the final two like this 

![image](https://user-images.githubusercontent.com/22151537/204726708-b0ca1143-e36d-493c-8748-23c61d7876c2.png)